admin/views/about.php

```diff
@@ -0,0 +1,219 @@
+			<h2><?php _e( 'More in this release', 'wordpress-seo' ); ?></h2>
+
+			<div class="feature-section col two-col">
+				<div>
+					<span class="dashicons dashicons-twitter"></span>
+					<h4><?php _e( 'Twitter Galleries', 'wordpress-seo' ); ?></h4>
+
+					<p>
+						<?php printf( __( 'If you use galleries in your posts and have Twitter meta data enabled on %1$sSocial → Twitter%2$s, galleries will now be auto-detected and put out as Gallery Cards. All that’s left for your is to %3$sverify the cards with Twitter%2$s.', 'wordpress-seo' ), '<a href="' . admin_url( 'admin.php?page=wpseo_social#top#twitterbox' ) . '">', '</a>', '<a target="_blank" href="http://yoa.st/twittercardsverify">' ); ?>
+					</p>
+				</div>
+
+				<div class="last-feature">
+					<span class="dashicons dashicons-performance"></span>
+					<h4><?php _e( 'Performance Improvements', 'wordpress-seo' ); ?></h4>
+
+					<p>
+						<?php _e( 'This release has made the code of WordPress SEO that runs on the frontend of your site approximately 30% faster.', 'wordpress-seo' ); ?>
+					</p>
+				</div>
+
+				<div>
+					<span class="dashicons dashicons-admin-generic"></span>
+					<h4><?php _e( 'Improved Settings Screens', 'wordpress-seo' ); ?> </h4>
+
+					<p>
+						<?php _e( 'All the settings screens have been updated to use exactly the same, tab-based, styling, making them much more comprehensible.', 'wordpress-seo' ); ?>
+					</p>
+				</div>
+
+				<div class="last-feature">
+					<span class="dashicons dashicons-translation"></span>
+					<h4><?php _e( 'More Translations', 'wordpress-seo' ); ?></h4>
+
+					<p>
+						<?php printf( __( 'WordPress SEO ships, at time of release, with 26 translations, of which 10 are complete. That\'s a huge improvement from last time, and we\'re improving every week. Join us at %1$stranslate.yoast.com%2$s!', 'wordpress-seo' ), '<a target="_blank" href="https://translate.yoast.com/projects/wordpress-seo">', '</a>' ); ?>
+					</p>
+				</div>
+			</div>
+		</div>
+
+		<hr>
+
+		<div class="return-to-dashboard">
+			<a href="<?php echo remove_query_arg( 'intro' ); ?>"><?php _e( 'Go to The General settings page →', 'wordpress-seo' ); ?></a>
+		</div>
+
+	</div>
+
+	<div id="credits" class="wpseotab">
+		<p class="about-description">
+			<?php
+				printf( __( 'While most of the development team is at %1$sYoast%2$s in the Netherlands, WordPress SEO is created by a worldwide team.', 'wordpress-seo' ), '<a target="_blank" href="https://yoast.com/">', '</a>' );
+				echo ' ';
+				printf( __( 'Want to help us develop? Read our %1$scontribution guidelines%2$s!', 'wordpress-seo' ), '<a target="_blank" href="http://yoa.st/wpseocontributionguidelines">', '</a>' );
+			?>
+		</p>
+
+		<h4 class="wp-people-group"><?php _e( 'Project Leaders', 'wordpress-seo' ); ?></h4>
+		<ul class="wp-people-group " id="wp-people-group-project-leaders">
+			<?php
+			$leaders = array(
+				'jdevalk'   => (object) array(
```

Comment:
Mostly just readability in the later code. Saves hassle of space code style commits :D